tests/baselines/reference/emptyArrayBindingPatternParameter04.js

```diff
@@ -0,0 +1,11 @@
+//// [emptyArrayBindingPatternParameter04.ts]
+
+
+function f([] = [1,2,3,4]) {
```

Comment:
The idea was simply to get a case where the parameter has an initializer.

Though I'm not seeing why this would be invalid; right it's perfectly valid to do something like

```TypeScript
let [a] = [1,2];
// or even
let [] = [1,2]
```

In `emptyArraybindingPatternParameter04.js`, you could be saying "I'm expecting an array; I don't care what the contents are, but if I didn't get one, I want this effect to take place."